packages/taco/test/conditions/context.test.ts

```diff
@@ -52,8 +57,8 @@ describe('context', () => {
 
   describe('reserved context parameters', () => {
     it.each([
-      [USER_ADDRESS_PARAM_DEFAULT, 'EIP4361'],
-      [USER_ADDRESS_PARAM_EXTERNAL_EIP4361, 'SSO4361'],
+      [USER_ADDRESS_PARAM_DEFAULT, EIP4361],
+      [USER_ADDRESS_PARAM_DEFAULT, SSO_EIP4361],
```

Comment:
Can we add EIP1271 here too to be complete.

```suggestion
      [USER_ADDRESS_PARAM_DEFAULT, SSO_EIP4361],
      [USER_ADDRESS_PARAM_DEFAULT, EIP1271],
```